tests/copp/copp_utils.py

```diff
@@ -37,6 +39,18 @@ def limit_policer(dut, pps_limit):
     dut.script(cmd="{} {} {} {}".format(_UPDATE_COPP_SCRIPT, pps_limit,
                                         _BASE_COPP_CONFIG, _TEMP_COPP_CONFIG))
     dut.command("docker cp {} {}".format(_TEMP_COPP_CONFIG, _SWSS_COPP_CONFIG))
+    # Comment out copp config generation in start.sh
```

Comment:
I would leave a quick note on why we need to do this in the comment, or even just a summary of the behavior (e.g. "disable copp config generation so the modified config isn't overwritten").